### PR TITLE
[MAINTENANCE] typing - `expect_column` - `to_exist` and `to_not_be_null`

### PR DESCRIPTION
**Pull request by Kilo59**
- Typing these 2 expectations to make a needed change as part of #8630  safer

------

- [x] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

**Comment by ghost** [comment]
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=8652&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

####

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=8652&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
CodeSee Map legend
</details>

File: great_expectations/expectations/core/expect_column_to_exist.py [hunk 1]
```diff
@@ -190,25 +196,24 @@ def _prescriptive_renderer(
...
-                        "params": params,
-                        "styling": styling,
-                    },
-                }
+                content_block_type="string_template",
+                string_template={
+                    "template": template_str,
+                    "params": params,
+                    "styling": styling,
+                },
```
Comment (wookasz) [inline great_expectations/expectations/core/expect_column_to_exist.py:204]: ❤️

**Review by wookasz — APPROVED** [review]
Nice! :shipit:

File: pyproject.toml [hunk 2]
```diff
@@ -87,7 +87,6 @@ exclude = [
     'expectations/core/expect_column_values_to_not_match_regex\.py',                                       # 2
     'expectations/core/expect_column_values_to_not_match_like_pattern_list\.py',                           # 3
     'expectations/core/expect_column_values_to_not_match_like_pattern\.py',                                # 2
-    'expectations/core/expect_column_values_to_not_be_null\.py',                                           # 2
     'expectations/core/expect_column_values_to_not_be_in_set\.py',                                         # 2
```
Comment (Shinnnyshinshin) [inline pyproject.toml:90]: 🚀

File: great_expectations/expectations/expectation.py [hunk 3]
```diff
@@ -2738,10 +2739,16 @@ class ColumnMapExpectation(BatchExpectation, ABC):
             kwargs from the Expectation Configuration.
     """
 
-    map_metric: Optional[str] = None
-    domain_keys = ("batch_id", "table", "column", "row_condition", "condition_parser")
```
Comment (Shinnnyshinshin) [inline great_expectations/expectations/expectation.py:2742]: ❤️

File: great_expectations/expectations/expectation.py [hunk 4]
```diff
@@ -327,25 +328,25 @@ class Expectation(metaclass=MetaExpectation):
         2. Data Docs rendering methods decorated with the @renderer decorator. See the
     """
 
-    version = ge_version
-    domain_keys: Tuple[str, ...] = ()
-    success_keys: Tuple[str, ...] = ()
-    runtime_keys: Tuple[str, ...] = (
+    version: ClassVar = ge_version
```
Comment (Shinnnyshinshin) [inline great_expectations/expectations/expectation.py:331]: This was a good opportunity to re-familiarize myself on `ClassVar` :) Thanks

**Review by Shinnnyshinshin — APPROVED** [review]
Looks good :) thank you sir

**Pull request merged**